src/main/java/teammates/ui/webapi/action/GetResponsesCountAction.java

```diff
@@ -0,0 +1,75 @@
+package teammates.ui.webapi.action;
+
+import java.util.List;
+
+import org.apache.http.HttpStatus;
+
+import teammates.common.datatransfer.FeedbackSessionResultsBundle;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.EntityNotFoundException;
+import teammates.common.util.Const;
+import teammates.ui.webapi.output.ResponsesCountData;
+
+/**
+ * Checks whether a session has responses.
+ */
+public class GetResponsesCountAction extends Action {
+
+    @Override
+    protected AuthType getMinAuthLevel() {
+        return AuthType.LOGGED_IN;
+    }
+
+    @Override
+    public void checkSpecificAccessControl() {
+        // Anyone can check whether a course has responses
+
+        String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        if (feedbackSessionName != null) {
+            String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+            FeedbackSessionAttributes feedbackSession = logic.getFeedbackSession(feedbackSessionName, courseId);
+
+            gateKeeper.verifyAccessible(
+                    logic.getInstructorForGoogleId(courseId, userInfo.getId()),
+                    feedbackSession);
+        }
+    }
+
+    @Override
+    public ActionResult execute() {
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        if (logic.getCourse(courseId) == null) {
+            return new JsonResult("No course with id: " + courseId, HttpStatus.SC_NOT_FOUND);
+        }
+
+        String feedbackQuestionID = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_ID);
+        if (feedbackQuestionID == null) {
+            List<FeedbackSessionAttributes> feedbackSessionAttributes = logic.getFeedbackSessionsForCourse(courseId);
```

Comment:
I hope you know there is a method called `Logic#hasResponsesForCourse`

Reply:
Yes, but that only returns true/false if there are responses or not? The API needs to return the quantity of responses...

Reply:
> The API needs to return the quantity of responses...

Do we really need that?